pepper-apis/src/main/java/org/broadinstitute/ddp/housekeeping/schedule/CheckKitsJob.java

```diff
@@ -54,25 +66,15 @@ public static void register(Scheduler scheduler, Config cfg) throws SchedulerExc
         scheduler.addJob(job, true);
         LOG.info("Added job {} to scheduler", getKey());
 
-        if (!cfg.getBoolean(ConfigFile.Kits.CHECK_ENABLED)) {
-            LOG.warn("Job {} is disabled, no trigger added", getKey());
-            return;
-        }
-
         int intervalSecs = cfg.getInt(ConfigFile.Kits.INTERVAL_SECS);
         Trigger trigger = TriggerBuilder.newTrigger()
                 .withIdentity(Keys.Kits.CheckTrigger)
                 .forJob(getKey())
                 .withSchedule(SimpleScheduleBuilder.repeatSecondlyForever(intervalSecs))
-                .startNow()
+                .startAt(new Date(Instant.now().toEpochMilli() + INITIAL_TRIGGER_DELAY_SECS * 1000))
```

Comment:
Added a bit of delay here to give Housekeeping time to finish booting up before running the job.